app/components/wokes/affiliation_row_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="mb-3 row">
```

Comment:
A nested form must have a row component that renders an individual form. This is for affiliations. It is a bit more complicated because it uses autocomplete.